pyaedt/modeler/Object3d.py

```diff
@@ -880,8 +955,13 @@ def part_coordinate_system(self, sCS):
 
     @property
     def solve_inside(self):
-        """Get or Set part solve inside flag as a boolean value
-        True if "solve-inside" is activated for the part
+        """Part solve inside flag as a Boolean value.
```

Comment:
```suggestion
        """Part solve inside flag.
```
Flags are generally boolean.